examples/plotting/histogram.py

```diff
@@ -16,16 +16,16 @@
 def make_plot(title, hist, edges, x, pdf, cdf):
     p = figure(title=title, tools='', background_fill_color="#fafafa")
     p.quad(top=hist, bottom=0, left=edges[:-1], right=edges[1:],
-           fill_color="navy", line_color="white", alpha=0.5)
-    p.line(x, pdf, line_color="#ff8888", line_width=4, alpha=0.7, legend_label="PDF")
-    p.line(x, cdf, line_color="orange", line_width=2, alpha=0.7, legend_label="CDF")
+           fill_color="#004488", line_color="#ffffff", alpha=0.5)
+    p.line(x, pdf, line_color="#BB5566", line_width=4, alpha=0.7, legend_label="PDF")
+    p.line(x, cdf, line_color="#DDAA33", line_width=4, alpha=0.7, legend_label="CDF")
```

Comment:
@robinokwanma which palette did these colors come from? Can you import the palette and pull the colors from the palette? That will help users understand how to use the palettes directly themselves. There is an illustration of this technique here;

https://github.com/bokeh/bokeh/pull/13013/files#diff-ec125147c9315e38cac7edcc43520e3d039110ae1ff13a3d36fbc5de544ce6d0R21